initfiles/bash/etc/init.d/install-init.in

```diff
@@ -184,9 +184,17 @@ echo ""
 
 # installing files
 installConfs "environment.conf" $configs/ 1 || exit 1
-installConfs "environment.xml" $configs/  1 || exit 1
 installConfs "genenvrules.conf" $configs/ 1 || exit 1
 
+echo "Generate environment.xml"
+# Generate environment.xml
+${path}/sbin/envgen -env ${path}/etc/HPCCSystems/rpmnew/environment.xml -ip . -roxienodes 1 -thornodes 1 -slavesPerNode 1 -o data4=/var/lib/[NAME]/hpcc-data4/[COMPONENT] -set_xpath_attrib_value Software/EclCCServerProcess/Option @name maxCompileThreads -set_xpath_attrib_value Software/EclCCServerProcess/Option @value 2 -override roxie,@allFilesDynamic,true -override roxie,@laxyOpen,smart -override roxie,@localSlave,true -override roxie,@slaveConfig,simple > /dev/null 2>&1 || exit 1
+cp ${path}/etc/HPCCSystems/rpmnew/environment.xml ${path}/testing/regress/environment.xml || exit 1
```

Comment:
What's the point of having a separate environment for the regression suite if it's always going to be the same as the standard one? I thought we used to have a different one because the settings appropriate for the regression suite were different?